zed_theme/src/lib.rs

```diff
@@ -1,3 +1,34 @@
-pub mod zed_theme_schema;
+mod zed_theme_schema;
 
 pub use zed_theme_schema::*;
+
+use std::collections::HashMap;
+
+const COMPILED_THEME_BYTES: &[u8] =
+    include_bytes!(concat!(env!("OUT_DIR"), "/compiled_themes.bin"));
+
+pub fn get_zed_themes() -> HashMap<String, ThemeContent> {
+    let decompressed_themes = miniz_oxide::inflate::decompress_to_vec(COMPILED_THEME_BYTES)
+        .expect("Compiled themes can't be decompressed?");
+    let theme_families: Vec<ThemeFamilyContent> =
+        serde_json_lenient::from_slice(&decompressed_themes)
+            .expect("Compiled themes aren't valid lenient JSON?");
+
+    let mut themes = HashMap::new();
+    for theme_family in theme_families {
+        for theme in theme_family.themes {
+            themes.insert(theme.name.clone(), theme);
+        }
+    }
+    themes
```

Comment:
Same here, this could be done using `.into_iter()` and `collect()`.

Reply:
Right, oop